public/assets/inst.js

```diff
@@ -137,12 +137,14 @@ function runInstruction(raw) { //, RISCV) {
         // I-TYPE, opcode: 0b0010011
         case 0x13:
             var funct3 = ((raw >>> 12) & 0x7);
+            RISCV.instruction_amounts['arithmetic'] = RISCV.instruction_amounts['arithmetic'] + 1;
             switch(funct3) {
                 
                 // ADDI
                 case 0x0:
                     RISCV.gen_reg[((raw >>> 7) & 0x1F)] = RISCV.gen_reg[((raw >>> 15) & 0x1F)].add(signExtLT32_64(((raw >> 20))));
                     RISCV.pc += 4;
+                    //RISCV.instruction_amounts.set('arithmetic', RISCV.instruction_amounts.get('arithmetic') + 1);
```

Comment:
how come you commented it out here?

Reply:
i can remove the comment; i moved it somewhere into the higher scope so it encompasses all the cases